addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6614,7 +6653,12 @@ msgctxt "#13297"
 msgid "Not connected. Check network settings."
 msgstr ""
 
-#empty string with id 13298
+#. Label of the button used to disconnect a game controller from the emulated game port
+#: xbmc/games/controllers/windows/ControllerSelect.cpp
+#: xbmc/games/controllers/windows/GUIPortList.cpp
+msgctxt "#13298"
+msgid "Disconnected"
```

Comment:
These and the following strings are not introduced by this PR. Rebase accident????

Reply:
Something goes strange wrong on one rebase and has added some commits where not relates to it.
Strange is if I create request as new they goes away and only my one stays in.